protecode/scanning.py

```diff
@@ -137,20 +138,24 @@ def iter_vulnerabilities_with_assessments(
                     continue
                 yield component, vulnerability, tuple(vulnerability.triages())
 
-        # only consider the most recent 50 products to import assessments from
-        # assumption: assessments are copied anyways from older assessments to
-        # newer ones so considering old assessments won't have any extra value
-        products_to_import_from.sort(
-            key=lambda p: p.product_id(),
-            reverse=True,
-        )
-        products_to_import_from = products_to_import_from[:50]
-
-        # do a shuffle to increase cache hits as otherwise same product ids are
-        # more likely to be handled simultaneously among the different threads
-        random.shuffle(products_to_import_from)
-
         for product in products_to_import_from:
+            if delete_inactive_products_after:
+                if deletion_after := product.custom_data().get('DELETION_AFTER'):
```

Comment:
naming-nit: how about calling this `delete_after` | `DELETE_AFTER`? to me, this sounds a bit more obvious (considering this is what the code will do (delete)) :-)